shared/searchv3/services-filter/index.js

```diff
@@ -118,4 +127,20 @@ class Filter extends Component<void, Props, {hoveredService: ?Constants.Service}
   }
 }
 
+const serviceTooltipStyle = {
+  ...transition('opacity'),
+  backgroundColor: globalColors.black_40,
+  borderBottom: '2px solid white',
```

Comment:
run this in RN and you'll see. RN has sep. values for this, not the short hand ones so you have to do
borderBottomColor: globalColors.white,
borderBottomWidth: 2, etc